### ISSUE TITLE
How to get the MAP?

### ISSUE THREAD

**Issue by skyneta** [zisianw/FaceBoxes.PyTorch#2]
Do you have the tool code to get your accuracy, can you provide it?Thanks！

**Comment by skyneta** [comment]
I use the https://github.com/xtanitfy/tf_ssd/blob/master/src/res_eval.py to do eval and the results as follow:
VOC07 metric? Yes
AP for face = 0.9024
Mean AP = 0.9024
~~~~~~~~
Results:
0.902
0.902
~~~~~~~~

**Comment by skyneta** [comment]
Then I eval the MTCNN that llink is https://github.com/AITTSMD/MTCNN-Tensorflow and results as such:
VOC07 metric? Yes
AP for face = 0.8966
Mean AP = 0.8966
~~~~~~~~
Results:
0.897
0.897
~~~~~~~~

**Comment by zisianw** [comment]
Hi @skyneta, we use [step 3](https://github.com/zisianw/FaceBoxes.PyTorch#evaluation) in our evaluation section to obtain the results. Additionally, to generate DiscROC.txt for FDDB, please refer to [their webpage](http://vis-www.cs.umass.edu/fddb/index.html).
